Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Driver.hs

```diff
@@ -1109,13 +1122,16 @@ addDriversInFleet merchantShortId opCity fleetOwnerId req = do
               }
       mobileNumberHash <- getDbHash req_.driverMobileNumber
       oldPerson <- QPerson.findByMobileNumberAndMerchantAndRole "+91" mobileNumberHash moc.merchantId DP.DRIVER
-      person <- case oldPerson of
-        Nothing -> DReg.createDriverWithDetails authData Nothing Nothing Nothing Nothing Nothing moc.merchantId moc.id True
-        Just p -> return p
-      checkAssoc <- B.runInReplica $ FDV.findByDriverIdAndFleetOwnerId person.id fleetOwnerId True
-      when (isJust checkAssoc) $ throwError (InvalidRequest "Driver already associated with fleet")
-      assoc <- FDA.makeFleetDriverAssociation person.id fleetOwnerId (DomainRC.convertTextToUTC (Just "2099-12-12"))
-      FDV.create assoc
+      case oldPerson of
+        Nothing -> void $ DReg.createDriverWithDetails authData Nothing Nothing Nothing Nothing Nothing moc.merchantId moc.id True
+        Just person_ -> do
+          fork "Sending Auth SMS for driver" $ do
+            dInfo <- QDriverInfo.findById person_.id >>= fromMaybeM DriverInfoNotFound
+            transporterConfig <- CTC.findByMerchantOpCityId moc.id Nothing >>= fromMaybeM (TransporterConfigNotFound moc.id.getId)
+            fleetOwner <- B.runInReplica $ QPerson.findById (Id fleetOwnerId :: Id DP.Person) >>= fromMaybeM (InvalidRequest "Fleet Owner not found")
+            case dInfo.verified of
```

Comment:
This case match to be replace with always `sendDeepLinkWithConsent and not Auth`